charts/hedera-mirror/templates/tests/pod.yaml

```
@@ -0,0 +1,34 @@
+{{- if .Values.test.enabled -}}
+apiVersion: v1
+kind: Pod
+metadata:
+  labels: {{- include "hedera-mirror.labels" . | nindent 4 }}
+  name: {{ include "hedera-mirror.fullname" . }}-acceptance-test
+  namespace: {{ include "hedera-mirror.namespace" . }}
+  annotations:
+    helm.sh/hook: test
+    helm.sh/hook-delete-policy: hook-failed,hook-succeeded
+spec:
+  containers:
+    - name: acceptance-tests
+      image: "{{ .Values.test.image.repository }}:{{ .Values.test.image.tag | default .Chart.AppVersion }}"
```

This should also use the `global.image.tag`. You'll need to add a `global.image: {}` to the values.yaml.
```suggestion
      image: "{{ .Values.test.image.repository }}:{{ .Values.global.image.tag | default .Values.test.image.tag | default .Chart.AppVersion }}"
```

---

Sure, added that.